core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -34,6 +34,7 @@ oppia.factory('StatsReportingService', [
       $http, StopwatchObjectFactory, messengerService,
       UrlInterpolationService, STATS_REPORTING_URLS, siteAnalyticsService) {
     var explorationId = null;
+    var explorationName = null;
```

Comment:
Actually can we use explorationTitle throughout (in the embedding script too), to preserve existing terminology?

Reply:
Yes, and done!